app/Models/WaitingRoom.php

```diff
@@ -30,6 +30,24 @@ public function users()
     }
 
     /**
+     * Gets a CSV export of all the users in this waiting room.
+     *
+     * @return \League\Csv\ $csv
+     */
+    public function getCSVExport()
```

Comment:
I agree it could be a helper, my only worry is im not sure down the road how long this feature will be used and what it will be exporting. EG: will the competitions export also show reportback progress? cause thats something which wont be inside the waiting rooms